test/unit/test_put_get.py

```diff
@@ -74,3 +79,19 @@ def test_put_error(tmpdir):
         sf_file_transfer_agent.result()
 
     chmod(file1, 0o700)
+
+
+def test_percentage(tmpdir):
+    """Tests for ProgressPercentage classes."""
+    assert 1.0 == percent(0, 0)
+    assert 1.0 == percent(20, 0)
+    assert 1.0 == percent(40, 20)
+    assert 0.5 == percent(14, 28)
+
+    file_path = tmpdir / 'zero_file1'
+    from pathlib import Path
+    Path(file_path).touch()
```

Comment:
Please take a look at https://docs.pytest.org/en/stable/tmpdir.html `tmpdir` and `tmp_path` are not the same.
```suggestion
def test_percentage(tmp_path):
    """Tests for ProgressPercentage classes."""
    assert 1.0 == percent(0, 0)
    assert 1.0 == percent(20, 0)
    assert 1.0 == percent(40, 20)
    assert 0.5 == percent(14, 28)

    file_path = tmpdir / 'zero_file1'
```